tools/gui/qtkontakt.py

```diff
@@ -155,11 +158,22 @@ def setup(self):
             # Benötigt wird: PLZ's der Impfzentren, Telefonnummer, Mail
             # Alles andere wird daher deaktiviert
             # !!!! wir erlauben aktuell alle eingaben, da diese später für die terminsuche benötigt werden. !!!
-            # self.readonly_alle_line_edits(("i_plz_impfzentren", "i_telefon", "i_mail"))
+            # 
             # self.i_code_impfzentren.setInputMask("")
```

Comment:
können wir ja dann auch alle löschen

Reply:
Ja, ich habe jetzt auch rausgenommen, dass der Code auf XXXX-XXXX-XXXX gesetzt wird. Das wird weder gespeichert noch is es editierbar.